local-running/src/main/java/uk/gov/di/ipv/coreback/handlers/JourneyEngineHandler.java

```diff
@@ -32,33 +32,9 @@ public class JourneyEngineHandler {
     public static final String FEATURE_SET = "feature-set";
     public static final String LANGUAGE = "language";
 
-    private final ProcessJourneyEventHandler processJourneyEventHandler;
```

Comment:
This solution of creating the classes _per call_ feels more appropriate, as closer matches the way a lambda would work. Have also checked and this does work